Wording changes (#5)
DataDog/datadog-static-analyzer-github-action: README.md

> | `dd_api_key` | Your Datadog API key. This key is created by your [Datadog organization][1] and should be stored as a [secret][2].         | True     |                 |
> | `dd_app_key` | Your Datadog application key. This key is created by your [Datadog organization][1] and should be stored as a [secret][2]. | True     |                 |
> | `dd_service` | The service you want your results tagged with.                                                                             | True     |                 |

Reviewer: ```suggestion
| `dd_api_key` | Your Datadog API key. This key is created by your [Datadog organization][1] and should be stored as a [secret][2].         | Yes     |
| `dd_app_key` | Your Datadog application key. This key is created by your [Datadog organization][1] and should be stored as a [secret][2]. | Yes     |
| `dd_service` | The service you want your results tagged with.                                                                             | Yes     |
```

| `dd_env`     | The environment you want your results tagged with. Datadog recommends using `ci` as the value for this input.              | False    | `none`          |

> | `dd_site`    | The [Datadog site][3] to send information to. You are currently viewing the documentation for the {{< region-param key="dd_site_name" code="true" >}} site, for which the correct value for this input would be {{< region-param key="dd_site" code="true" >}}                                                                          | False    | `datadoghq.com` |

Reviewer: ```suggestion
| `dd_site`    | The [Datadog site][3] to send information to. Your Datadog site is: {{< region-param key="dd_site" code="true" >}}.                                                                          | False    | {{< region-param key="dd_site" code="true" >}}   |
```
